docs/resources/samples/excel-samples.md

```diff
@@ -50,6 +50,99 @@ function main(workbook: ExcelScript.Workbook) {
 }
 ```
 
+### Add data to a range
+
+This script adds a set of values to a new worksheet. The values start in cell **A1**. The data used in this script is pre-defined, but could be sourced from other places in or out of the workbook.
```

Comment:
Per AHD, seems this should be [predefined](https://www.ahdictionary.com/word/search.html?q=predefined)
```suggestion
This script adds a set of values to a new worksheet. The values start in cell **A1**. The data used in this script is predefined, but could be sourced from other places in or out of the workbook.
```